web-components/src/components/draggable/scss/draggable-item.scss

```diff
@@ -0,0 +1,16 @@
+:host {
+  display: block;
+}
+
+.md-draggable-row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  align-content: flex-start;
+  border: 1px solid rgb(177, 33, 33);
```

Comment:
please refer to the color token for border instead of hard coding an rgb value

Reply:
Done.